openfisca_france/parameters/bourses_superieur/criteres_sociaux.yaml

```diff
@@ -0,0 +1,26 @@
+nombre_mensualites:
+  description: Nombre de mois de versement d'une bourse de l'enseignement supérieur sur critères sociaux
+  reference: https://www.legifrance.gouv.fr/loda/id/JORFTEXT000042170743/
+  values:
```

Comment:
```suggestion
  reference: https://www.legifrance.gouv.fr/loda/id/JORFTEXT000042170743/
  unit: month
  values:
```

Reply:
🤔 Intégré après avoir vu que cette unité était déjà utilisée par openfisca-france dans [aide_mobilite_internationale/duree_sejour](openfisca_france/parameters/prestations/aide_mobilite_internationale/duree_sejour.yaml) et nous en avons besoin ici.
Néanmoins, cela sort de la liste de valeurs d'unités indiquées dans la [documentation](https://openfisca.org/doc/coding-the-legislation/legislation_parameters.html#how-to-write-a-new-parameter) (pour la syntaxe plus récente de paramètres que celle employée dans openfisca-france où `unit` est sous `metadata`.